src/main/java/io/github/skyblockcore/util/APIUtils.java

```diff
@@ -0,0 +1,71 @@
+package io.github.skyblockcore.util;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class APIUtils {
+
+    private static JsonElement PlayerInfo;
+    private static String Player;
+
+    private static HashMap<String, String> PlayerData = new HashMap<String, String>();
+    public static String getUsername(String uuid) {
+        if (PlayerData.containsValue(uuid)) {
+            for(Map.Entry<String, String> entry : PlayerData.entrySet()){
+                if(Objects.equals(entry.getValue(), uuid)) {
+                    Player = entry.getKey();
+                }
+            }
```

Comment:
[Please use HashMap#get](https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html#get-java.lang.Object-)

Reply:
lmao

Reply:
it was my 5 am brain ok? i forgot that exsits